features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -194,7 +196,8 @@ Feature: Collection
   # another person signs up
   When I log out
     And I am logged in as "myname2"
-  When I go to the collections page
+  When all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.